scripts/migration/renamings.json5

```diff
@@ -1478,5 +1478,36 @@
         },
       },
     },
+    {
+      oldName: 'Blockly.Input',
+      newName: 'Blockly.inputs.Input',
+      exports: {
+        'Align': {
+          oldPath: 'Blockly.Input.Align',
+          newPath: 'Blockly.inputs.Align',
+        },
+      },
+    },
+    {
+      oldName: 'Blockly.Align',
+      newName: 'Blockly.inputs.Align',
+    },
+    {
+      oldName: 'Blockly',
+      exports: {
+        'ALIGN_LEFT': {
+          newPath: 'Blockly.inputs.Align.LEFT',
+        },
+        'ALIGN_CENTER': {
+          newPath: 'Blockly.inputs.Align.CENTER',
+        },
+        'ALIGN_RIGHT': {
+          newPath: 'Blockly.inputs.Align.RIGHT',
+        },
+        'inputTypes': {
+          newModule: 'Blockly.inputs',
+        },
+      },
+    },
```

Comment:
@cpcallen can you check these renamings for me as well?

Reply:
@BeksOmega: I do not see any change to the `goog.declareModueId()` calls, so none of these should have a `newName` at the module level—only new paths.

I will send you a PR to fix.

Reply:
(Oh, this has not been merged yet, so I will add a commit.)